R/data.R

```diff
@@ -0,0 +1,17 @@
+#' Antidepressant trial data.
+#'
+#' A dataset containing the data from a public available antidepressant clinical trial of an active drug versus placebo.
```

Comment:
I'm not entirely sure but perhaps:

A dataset containing ~the~ data from a public a

Reply:
done